include/boost/math/fft/real_algorithms.hpp

```diff
@@ -487,6 +487,137 @@
       real_dft_composite_outofplace(in_first,in_last,out,sign,alloc);
   }
   
+  
+  template <class T, class allocator_t>
+  void real_inverse_dft_composite_outofplace(
```

Comment:
The halfcomplex-to-real is again the reverse operation of the real-to-halfcomplex.